src/dns-simple-server.toit

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by an MIT-style license that can be
 // found in the LICENSE file.
 
-import binary show BIG-ENDIAN
-import bytes show Buffer
+import io show BIG-ENDIAN
```

Comment:
Do we still use this?

Reply:
I had to update the `get` function which was still using it, but it's not used anymore.
done.